src/main/resources/templates/db_scripts/schema.json

```diff
@@ -9,7 +9,7 @@
     },
     {
       "tableName":"feefines",
-      "fromModuleVersion":"15.7.1",
+      "fromModuleVersion":"15.10.0",
       "withMetadata":true,
       "customSnippetPath": "populate-feefines.sql",
```

Comment:
I didn't think we changed the fromModuleVersion of a table for a configuration change. 

Is this so that the snippet (which has been extended) can be triggered rather than an explicit upgrade script?

Reply:
Yes, the snippet won't be executed if we don't change the version. But you're right, I'll create a separate script.

Reply:
I think the confusion stems from our tooling having two different ways to do this and no clear guidance as to which to use